vms/platformvm/state/state.go

```diff
@@ -1326,10 +1323,6 @@ func (s *state) GetStartTime(nodeID ids.NodeID) (time.Time, error) {
 	return staker.StartTime, nil
 }
 
-func (s *state) GetEtnaHeight() (uint64, error) {
-	return database.GetUInt64(s.singletonDB, EtnaHeightKey)
```

Comment:
Can we delete `EtnaHeightKey`?

Reply:
I wasn't sure looking at it initially since IIUC, existing nodes will still have that value in their state and new nodes will not. I guess there isn't any worry of mistakingly re-introducing that state key in the future though, even if we remove it from source.